### PR TITLE
Add pyFG as requirement instead of note

### PR DESCRIPTION
**Pull request by userimack**
##### SUMMARY

Add pyFg as requirement instead of notes.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

Module fortios_config and fortios_ipv4_policy in fortios network module
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by caphrim007** [comment]
if this is fortios specific, then please remove the changes to bigip modules

**Comment by wojtek0806** [comment]
I concur, please remove the changes to f5 modules

File: lib/ansible/modules/notification/pushbullet.py [hunk 1]
```diff
@@ -53,6 +53,8 @@
    - Requires pushbullet.py Python package on the remote host.
      You can install it via pip with ($ pip install pushbullet.py).
      See U(https://github.com/randomchars/pushbullet.py)
+requirements:
+   - pushbullet.py
```
Comment (Akasurde) [inline lib/ansible/modules/notification/pushbullet.py]: Remove this as Line 19 already states this as a requirement.

**Comment by userimack** [comment]
I have modified the commit according to the comments. Please have a look once again.

**Comment by Akasurde** [comment]
cc @acozine could you please take a look ? LGTM

**Comment by caphrim007** [comment]
lgtm

**Comment by acozine** [comment]
Looks good, thank you @userimack.

**Pull request merged**